app.py

```diff
@@ -128,12 +128,15 @@
 PUNCTRE_LABEL = re.compile(r'.*-p$')
 INITIAL_PUNCT_LABELS = {'LRB-p', '[-p', '{-p'}
 
-def is_possible_punct_sequence(token):
-	return re.match(PUNCTRE, token) or token in [e['ptree_token'] for e in PUNCT_ESCAPING]
+def is_punct_postag(tag):
+	return tag in PUNCT_TAGS.values() or tag == SYMBOL_TAG
 
 def is_punct_label(label):
 	return re.match(PUNCTRE_LABEL, label) or label in [e['ptree_label'] for e in PUNCT_ESCAPING]
 
+def is_possible_punct_token(token):
+	return re.match(PUNCTRE, token) or token in PUNCT_TAGS.keys() or token in [i['ptree_token'] for i in PUNCT_ESCAPING]
```

Comment:
technically `.keys()` is redundant, more idiomatic to check whether something is a dict key without it

Reply:
https://github.com/nschneid/activedop/pull/59/commits/e85ae2e10518b367061b7299b7338fd62f719a71